libchisel/src/lib.rs

```diff
@@ -128,6 +134,12 @@ mod tests {
         assert!(result.is_ok());
     }
 
+    #[test]
+    fn from_error() {
+        let err: ModuleError = "custom message".to_string().into();
```

Comment:
Not really useful in this form, but is when trying to unwrap from a function returning `Result<..., String>` into `Result<..., ModuleError>`